package.json

```diff
@@ -2,6 +2,7 @@
   "name": "hp-admin",
   "version": "0.11.11",
   "private": true,
+  "homepage": "https://holo-host.github.io/holofuel-static-ui/",
```

Comment:
Wouldn't it default to the repo name, ie `https://holo-host.github.io/hp-admin/`?  Are you able to name the url to any arbitrary string?

Reply:
Also, if we can update the name, can we just call it: `https://holo-host.github.io/holofuel-ui/` to match the app name itself?

Reply:
it's on its own repo, not `hp-admin`. We can't call it `holofuel-ui` because that repo is already taken.

Just a reminder that users will just see `holofuel.holo.host` as the url